esprima.js

```diff
@@ -205,7 +214,12 @@
         DuplicateProtoProperty: 'Duplicate __proto__ fields are not allowed in object literals',
         ConstructorSpecialMethod: 'Class constructor may not be an accessor',
         DuplicateConstructor: 'A class may only have one constructor',
-        StaticPrototype: 'Classes may not have static property named prototype'
+        StaticPrototype: 'Classes may not have static property named prototype',
+        MissingFromClause: 'Missing from clause',
```

Comment:
Any time we update an error message, it may risk breaking (because some projects might rely on the message). However, moving from the generic "unexpected token" is considered less problematic because it can be perceived as an **upgrade** of the mesage.